core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2817,6 +2809,14 @@ public String toString() {
               + "to commit results.")
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_FILE_PERSISTENCE_INITIAL_WAIT_TIME =
```

Comment:
Should we add an option to our persist cli which let's them specify the wait time?

Reply:
I am considering either we can add a persist option like `forcePersist` which will use 0 wait time. Or we directly use 0 wait time. I think when the user call persist, they want to persist as soon as possible.

Reply:
By default the wait time should be zero, I just meant to provide an option in the CLI so the user does not need to pass something like `-Dalluxio.user.file...`

Reply:
I agree, the persist CLI should have a persist wait time option. if it's not provided, we directly use the default initial wait time value.